### PR TITLE
[zh] Resync configure probes

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by jihoon-seo** [comment]
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/622d5be7bdc865000830bd58#L77).
> content/zh/examples/pods/probe/grpc-liveness.yaml: no such file or directory

@tengqm PTAL!

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -333,6 +334,98 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+都被认作是探测失败，这一点与 HTTP 和 TCP 探测器类似。
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/probe/grpc-liveness.yaml
+```
+
+<!--
+After 15 seconds, view Pod events to verify that the liveness check has not failed:
+-->
+15 秒钟之后，查看 Pod 时间确认活跃性检查并未失败：
```
Comment (mengjiao-liu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:394]:
```suggestion
15 秒钟之后，查看 Pod 事件以确认活跃性检查并未失败：
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -333,6 +334,98 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+
+<!--
+Before Kubernetes 1.23, gRPC health probes were often implemented using [grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/),
+as described in the blog post [Health checking gRPC servers on Kubernetes](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/).
+The built-in gRPC probes behavior is similar to one implemented by grpc-health-probe.
+When migrating from grpc-health-probe to built-in probes, remember the following differences:
+-->
+在 Kubernetes 1.23 之前，gRPC 健康探测通常使用
+[grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/)
+来实现，如博客[Health checking gRPC servers on Kubernetes（对 Kubernetes 上的 gRPC 服务器执行健康检查）](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/)所描述。
```
Comment (mengjiao-liu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:408]:
```suggestion
来实现，如博客 [Health checking gRPC servers on Kubernetes（对 Kubernetes 上的 gRPC 服务器执行健康检查）](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/)所描述。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -471,10 +566,10 @@ Readiness and liveness probes can be used in parallel for the same container.
 Using both can ensure that traffic does not reach a container that is not ready
 for it, and that containers are restarted when they fail.
 -->
-HTTP 和 TCP 的就绪探测器配置也和存活探测器的配置一样的。
+HTTP 和 TCP 的就绪探测器配置也和存活探测器的配置完全相同。
 
 就绪和存活探测可以在同一个容器上并行使用。
-两者都用可以确保流量不会发给还没有准备好的容器，并且容器会在它们失败的时候被重新启动。
+两者都用可以确保流量不会发给还未就绪的容器，当这些探测失败时容器会被重新启动。
```
Comment (mengjiao-liu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:572]:
```suggestion
两者都可以确保流量不会发给还未就绪的容器，当这些探测失败时容器会被重新启动。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 4]
```diff
@@ -443,8 +537,8 @@ Readiness probes runs on the container during its whole lifecycle.
 Liveness probes *do not* wait for readiness probes to succeed. If you want to wait before executing a liveness probe you should use initialDelaySeconds or a startupProbe.
 -->
 {{< caution >}}
-活跃性探测器 *不等待* 就绪性探测器成功。
-如果要在执行活跃性探测器之前等待，应该使用 initialDelaySeconds 或 startupProbe。
+活跃性探测器 **不等待** 就绪性探测器成功。
+如果要在执行活跃性探测器之前等待，应该使用 `initialDelaySeconds` 或 `startupProbe`。
```
Comment (howieyuen) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:541]:
```suggestion
如果要在执行活跃探测器之前等待，应该使用 `initialDelaySeconds` 或 `startupProbe`。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Pull request merged**